src/components/pages/rules/flyouts/ruleDelete/deleteRule.scss

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/themes';
+@import 'src/styles/mixins';
+
+.delete-rule-flyout-container {
+
+  .padded-top-bottom {
+    margin-left: 0px;
+    @include rem-fallback(padding, 24px, 0px);
+  }
+
+  .delete-info {
+    display: flex;
+
+    .asterisk-svg {
+      align-self: flex-start;
+
+      svg {
+        @include square-px-rem(12px);
+        @include rem-fallback(margin-right, 6px);
+        @include rem-fallback(margin-top, 2px);
+      }
+    }
+  }
+
+  .delete-confirmation-text {
+    display: flex;
+
+    .check-svg {
+      align-self: flex-start;
+
+      svg {
+        @include square-px-rem(14px);
+        @include rem-fallback(margin-left, 14px);
+      }
+    }
+  }
+
+  .rule-details {
+    border-top-style: none;
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+    @include rem-fallback(padding-top, 10px);
+    @include rem-fallback(padding-bottom, 24px);
+  }
+
+  .rule-delete-error {
+    border: 1px transparent solid;
+    @include rem-fallback(margin, 12px, 0px);
+    @include rem-fallback(padding, 16px);
+  }
+
+  .delete-info, .delete-confirmation {
+    @include rem-fallback(font-size, 14px);
+    @include rem-fallback(padding-top, 24px);
+  }
+
+  @include themify($themes) {
+
+    .delete-info a,
+    .delete-confirmation a {
```

Comment:
single line
`.delete-confirmation a { color: themed('colorFlyoutConfirmationText'); }`

Reply:
it's has two selectors, so I think it should be on two lines

Reply:
The reason for placing each selector on its own line is to allow the reader to quickly understand how many selectors are associated with a given rule. The property styling exist for a similar purpose. All SASS rules are biased towards conciseness. So the justification for the selector styling doesn't imply anything for the property declaration styling. This is and has been the SASS styling throughout the repo to this point as defined in the [style guide](https://github.com/Azure/pcs-remote-monitoring-webui/wiki/Code-Guide#single-vs-multiple-declarations).

Reply:
I was confused because in the rule directly below, you didn't say to put it on one line and it's the same case (and the style guide is a little ambiguous for this specific scenario). I will fix both

Reply:
Cool. Yeah I just didn't want to spam every case with a comment :smiley:.